librz/il/definitions/common.h

```diff
@@ -0,0 +1,12 @@
+// SPDX-FileCopyrightText: 2021 heersin <teablearcher@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#ifndef BUILD_COMMON_H
+#define BUILD_COMMON_H
+
+#include "rz_types.h"
+
+typedef char *string;
+typedef bool bit;
```

Comment:
why these?

Reply:
We could remove them. Not necessary anymore